docs/guidelines/secret-scanning.md

```diff
@@ -0,0 +1,118 @@
+# Secret Scanning
+
+As developers, we know that secrets like passwords, API keys, and access tokens are critical to our work. But what happens when these secrets accidentally end up in our code, logs or error messages?
+
+This guideline aims to present relevant tools, some good practices for managing this risk, and what to do when we have messed up. The AppSec team provides a 4 hour hands-on workshop to get started with secret scanning.
+
+- [Recommended tools](#recommended-tools)
+- [Where to scan for secrets in our SDLC](#where-to-scan-for-secrets-in-our-sdlc)
+- [What to do when we have messed up](#what-to-do-when-we-have-messed-up)
+- [How do we manage secrets in our dev environments?](#how-do-we-manage-secrets-in-our-dev-environments)
+
+## Recommended tools
+
+- Snyk Code (in IDE)
+- [Trufflehog](https://github.com/trufflesecurity/trufflehog) (elsewhere)
+
+Please note that:
+
+- False positives is a sizeable challenge, scanners need to be fine tuned to your context (detectors, testing, verification +++)
+- Incremental scans would be helpful, you may not want to scan everything all the time
+- Don't underestimate the cultural change when changing how you work. Good practices are to document how your team does Secure Development and to threat model how you work. Use/revisit this information regularly in on-boarding of new team members and in retrospectives.
+
+## Where to scan for secrets in our SDLC
+
+The general idea is to catch secrets in code (and other places) as early as possible. Our recommendation for most teams is:
+
+While your are developing, in your development environment
+
+- Scan in your IDE (using Snyk Code)
+- Scan in your local build process (using Trufflehog)
+- Scan in git pre-commits (using Trufflehog)
+
+In your CI pipeline
+
+- Scan as part of PR checks (using Trufflehog)
+- Scan container images as well (using Trufflehog)
+
+Other places:
+
+- Scanning logs (using Trufflehog)
+
+## What to do when we have messed up
```

Comment:
This section should be highlighted to show important content i.e have a different formatting than the regular sections so its easily distinguishable.

Reply:
I'll add a few emoji's - let's see how the can make a difference